decentraland/unity-explorer
# fix: backpack info panel always showing last clicked item between lifecycles

## What does this PR change?



Fix #1982 

Restore original panel activation status every time the backpack panel get closed in order to avoid the persistence of the FullPanel content

## How to test the changes?



1. Launch the explorer
2. Open the backpack
3. Select a wearable
4. Check the info panel showing the clicked item info
5. Close the backpack
6. Open the backpack -> the info panel is empty
7. The same interaction should work the same between account changes (logout/login) as described in the issue

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
